mne/decoding/transformer.py

```diff
@@ -715,3 +717,164 @@ def transform(self, X):
         X = self.estimator.transform(X)
         X = np.reshape(X.T, [-1, n_epochs, n_times]).transpose([1, 0, 2])
         return X
+
+
+class Filterer(TransformerMixin):
+    """Estimator to filter RtEpochs
```

Comment:
ok thanks. The new decoding estimators should only support numpy arrays.

@kaichogami don't deprecate FilterEstimator for now, so that this kind of use case remains covered.